foundation_cms/blocks/timely_activations_cards_block.py

```diff
@@ -31,24 +30,13 @@ class TimelyActivationsCardsBlock(BaseBlock):
 
     title = blocks.CharBlock(required=False, help_text="Title for the activations section")
```

Comment:
Let's remove this field. We've decided to remove `title` field from any blocks to make our blocks more versatile.